compiler/modules/nback.llvm/llvm.bal

```diff
@@ -87,6 +89,29 @@ public class Module {
         return fn;
     }
 
+    // Corresponds to LLVMGetIntrinsicDeclaration
+    public function getIntrinsicDeclaration(IntrinsicFunctionName name) returns FunctionDecl {
+        StructType overflowArithmeticReturnType = structType(["i64", "i1"]);
+        match name {
+            "sadd.with.overflow.i64" => {
+                FunctionDecl fn = new ("llvm.sadd.with.overflow.i64", {returnType:overflowArithmeticReturnType, paramTypes:["i64","i64"]});
+                self.functionDeclns.push(fn);
+                return fn;
+            }
+            "ssub.with.overflow.i64" => {
+                FunctionDecl fn = new ("llvm.ssub.with.overflow.i64", {returnType: overflowArithmeticReturnType, paramTypes: ["i64", "i64"]});
+                self.functionDeclns.push(fn);
+                return fn;
+            }
+            "smul.with.overflow.i64" => {
+                FunctionDecl fn = new ("llvm.smul.with.overflow.i64", {returnType: overflowArithmeticReturnType, paramTypes: ["i64", "i64"]});
+                self.functionDeclns.push(fn);
+                return fn;
+            }
+        }
+        panic error ("Unknown intrinsic function");
```

Comment:
This is unreachable, so use `err:unreached`

Reply:
Fixed